src/plugins/nftModule/NftClient.ts

```diff
@@ -125,13 +125,13 @@ export class NftClient {
   }
 
   findMintWithMetadataByMetadata(
-    metadataAddress: PublicKey,
+    address: PublicKey,
     options?: Omit<FindMintWithMetadataByMetadataInput, 'address'>
```

Comment:
options required **metadataAddress** since omit was for **address**

Changed **metadataAddress** to **address**. This also complies with file conventions.